src/main/java/Duke.java

```diff
@@ -1,10 +1,144 @@
-public class Duke {
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+
+/**
+ * A chat bot reacting on tasks.
+ */
+
+public class Duke extends Application{
+    private static Storage storage;
+    private static TaskList tasks;
+    private static UI ui;
+    private static Parser newParser;
+
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+
+    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        storage.createDirectory("FileSaver1");
+        storage.updateList(tasks);
+        newParser.run(tasks, ui);
+        storage.updateDirectory(tasks);
+        tasks.showList();
+        System.out.println("Thank you for using! See you next time!");
     }
+
+    @Override
+    public void start(Stage stage) {
```

Comment:
This method seems to be a little long and goes beyond 30 LOC. Would be good to provide a javaDoc comment or shorten the code